jdaviz/core/events.py

```diff
@@ -255,13 +255,6 @@ def selected_index(self):
     def shared_image(self):
         return self._shared_image
 
-
-class SliceSelectWavelengthMessage(Message):
-    '''Message generated by the select slice plot plugin which is processed by the cubeviz helper'''
-    def __init__(self, wavelength=None, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self._wavelength = wavelength
-
     @property
     def wavelength(self):
         return self._wavelength
```

Comment:
this needs to go too (or it will become a part of `TableClickMessage`)

Reply:
Whoops, thanks for the catch!